plugins/chain_interface/include/eosio/chain/signals_processor.hpp

```diff
@@ -11,66 +11,60 @@ namespace eosio::chain {
 class signals_processor {
```

Comment:
I don't see this adding value anymore, I think it should just be removed and wire to the signals directly to trx finality status processor.

Reply:
Do make sure to move over the `FC_LOG_AND_DROP`

Reply:
Addressed:  9d87fcbe4a4f2214c0260780d3d2a512d5326cbc